### PR TITLE
COMMERCE-1285 added localisation on cards, compare and account manage…

### PR DESCRIPTION
**Pull request by fabio-d-m**
…r placeholder

@marco-leo I've added the key on the localisation files for the minicompare and the minium card.
I've also used {msg} for the placeholder on the account manager but now the strings should be translated

### CODE REVIEW

File: commerce-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -394,6 +394,7 @@ retry=Retry
 rounding-mode=Rounding Mode
 rounding-type=Rounding Type
 savings=Savings
+search-order=Search order…
```
Comment (ethanbustad) [inline commerce-lang/src/main/resources/content/Language.properties]: should have a capital `O` -- `Search Order...`. Language values should (almost) always be [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case) if a full sentence or [title case](https://en.wikipedia.org/wiki/Letter_case#Title_case) if not.

**Review by ethanbustad — CHANGES_REQUESTED** [review]
@FabioDiegoMastrorilli can you fix the lang value, and separate the `buildLang` files into a separate commit?

File: commerce-lang/src/main/resources/content/Language.properties (hunk 1 above)
Comment (fabio-d-m) [reply commerce-lang/src/main/resources/content/Language.properties]: Done it!

**Review by ethanbustad — APPROVED** [review]

**Comment by ethanbustad** [comment]
ci:test

**Comment by ethanbustad** [comment]
ci:test

**Comment by ethanbustad** [comment]
ci:test
Tests may pass now that https://github.com/liferay/com-liferay-commerce/pull/1436 is merged in.

**Pull request merged**